worker/src/util/adoc2html.ts

```diff
@@ -300,7 +300,7 @@ class FranklinConverter implements AdocTypes.Converter {
       },
       inline_image: (node) => this.templates.image(node as AdocTypes.AbstractBlock),
       image: (node) => {
-        const src = node.getAttribute('target') as string | undefined;
+        const src = node.getImageUri(node.getAttribute('target', '') as string);
```

Comment:
maybe worth watching out for image dirs pointing to a path that includes the `/_graphics` prefix, since we add it blindly on L308

Reply:
Thanks, @maxakuru. I tested this already, and the hard-coded `_graphics` prefix doesn't get in the way of the `imagesdir` attribute. Effectively, the current code constructs the path to the book's directory, and then tacks on the `_graphics` prefix. And then the `imagesdir` attribute, if used, is relative to this directory.